collector/collector.go

```diff
@@ -77,9 +77,14 @@ func (e *Exporters) SetVTYSHTimeout(timeout time.Duration) {
 	vtyshTimeout = timeout
 }
 
+// SetVTYSHPathspace sets the frr config path prefix (-N option for vtysh)
+func (e *Exporters) SetVTYSHPathspace(ns string) {
```

Comment:
not sure if `ns` variable name is relevant here, maybe just `s string`. would be better, otherwise it get confusing namespaces vs pathspaces

Reply:
Thanks, fixed that.